src/components/templates/Playa/Playa.tsx

```diff
@@ -128,7 +127,12 @@ const isPlaced = (venue: AnyVenue) => {
 const minZoom = () => (window.innerWidth - 2 * PLAYA_MARGIN_X) / PLAYA_WIDTH;
 
 const Playa = () => {
+  // @debt This will currently load all venues in firebase into memory.. not very efficient
   useFirestoreConnect("venues");
+  const venues = useSelector(orderedVenuesSelector);
```

Comment:
[note, ?OOS?] This is a legacy template that is scheduled to be removed, so shouldn't be an issue. We can probably even comment this out to be certain about it not getting run if we want.

Reply:
OOS, will handle this in a followup PR.